demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -562,3 +563,18 @@ def is_valid_display_name(self):
                 print_error(Errors.invalid_v2_integration_name(self.file_path))
                 return False
             return True
+
+    def is_all_params_not_hidden(self):
+        # type: () -> bool
+        """
+        Verify there are no hidden integration parameters.
+        Returns:
+            bool. True if there aren't hidden parameters False otherwise.
+        """
+        ans = True
```

Comment:
there is no need in the var(know it is common in this code)
just return False if you found hidden and True if you got to the end.

Reply:
I did it that way because I wanted to finish printing regarding all hidden parameters before exiting the function